mysql/datadog_checks/mysql/statements.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import copy
+import logging
+from contextlib import closing
+
+import pymysql
+
+from datadog_checks.base.utils.db.sql import compute_sql_signature
+from datadog_checks.base.utils.db.statement_metrics import StatementMetrics, apply_row_limits
+
+try:
+    import datadog_agent
+except ImportError:
+    from ..stubs import datadog_agent
+
+
+logger = logging.getLogger(__name__)
+
+
+METRICS = {
+    'count': 'mysql.queries.count',
+    'errors': 'mysql.queries.errors',
+    'time': 'mysql.queries.time',
+    'select_scan': 'mysql.queries.select_scan',
+    'select_full_join': 'mysql.queries.select_full_join',
+    'no_index_used': 'mysql.queries.no_index_used',
+    'no_good_index_used': 'mysql.queries.no_good_index_used',
+    'lock_time': 'mysql.queries.lock_time',
+    'rows_affected': 'mysql.queries.rows_affected',
+    'rows_sent': 'mysql.queries.rows_sent',
+    'rows_examined': 'mysql.queries.rows_examined',
+}
+
+DEFAULT_METRIC_LIMITS = {k: (10000, 10000) for k in METRICS.keys()}
```

Comment:
```suggestion
DEFAULT_STATEMENT_METRIC_LIMITS = {k: (10000, 10000) for k in STATEMENT_METRICS.keys()}
```
Is it possible to rename it like so it's more explicit?